### DISCUSSION TITLE
How to host and embed audio (`.mp3`, etc.)?

### DISCUSSION THREAD

**Discussion by HashaRock** [squidfunk/mkdocs-material#6184] [General]
Title.

Basically, my situation is pretty complex. I have around 15GB of audio files that I am trying to add on to my MKDocs but the issue is how I can incorporate these while keeping the site static and lowest cost as possible.

Each file is around 10-15 MB but with 1000s of files, which I am trying to add to the documentation. What would be the cheapest way to incorporate these and what would y'all recommend for organizing so many of these files to all be accessible for the docs?

Thanks in advance!

I already have these files on Mega but am willing to move them around as long as I can host them successfully and cleanly!

Edit: Also looking for fastest deployment options and with supporting such files!

**Comment by alexvoss** [comment]
What do you imagine the level of usage will be? Have you tried calculating the cost of hosting them in a cloud storage service? 

Also, what is the nature of the files? Are they of public interest so you could get some org to host them for you?

**Reply by HashaRock** [reply]
Shouldn’t be any usage which is crazy, probably around 1000 listens/dls a month. That should work with most cloud storage services but I simply don’t see a way to implement that with the markdown on the wiki

**Reply by alexvoss** [reply]
Whatever hosting you go with, you will probably need to include some HTML in the pages - unless you find a plugin that produces this for you from some bit of Markdown syntax. This [does exist for video at least](https://pypi.org/project/mkdocs-video/).
